osu.Game/Skinning/LegacySkin.cs

```diff
@@ -57,10 +59,22 @@ public override Drawable GetDrawableComponent(string componentName)
                     break;
             }
 
-            var texture = GetTexture(componentName);
+            float ratio = 0.72f; // brings sizing roughly in-line with stable

```

Comment:
Explain the reason for this again? Is this because some skins have transparency padding?

If so I'd rather this wasn't here, because it makes it even harder for people to know how much transparency to remove from their skins.

Reply:
No, this does what it says. It brings skin elements in-line with lazer (tested using default skin template and a few custom skins).

Reply:
Skinning is not even fully implemented yet - it's maybe implemented in like 4 drawable components, hardly enough to get a reasonable idea about this.

A better fix would be to alter the size of lazer's components to match stable's, no?

Reply:
The scale applies to every skinnable element, equally.

Reply:
I'll trust your judgement that this is close enough.